CppExport/src/exporter/CppExporter.cpp

```diff
@@ -136,7 +136,7 @@ Export::FragmentLayouter CppExporter::layouter()
 						  | Export::FragmentLayouter::NewLineBeforePostfix, "{", "\n", "}");
 	result.addRule("space", Export::FragmentLayouter::SpaceAtEnd, "", " ", "");
 	result.addRule("comma", Export::FragmentLayouter::SpaceAfterSeparator, "", ",", "");
-	result.addRule("baseClasses", Export::FragmentLayouter::SpaceAfterSeparator, "public: ", ",", "");
+	result.addRule("baseClasses", Export::FragmentLayouter::SpaceAfterSeparator, " : public ", ",", "");
```

Comment:
Not all inheritance is public in C++. You don't need to fix this now, but you should start a list of things that we don't yet support in Envision and which we'll need to implement. Here is the first entry:
- Inheritance modifiers (private, public, virtual ...)